File: julia/p_038.jl
using ProjectEuler
question(38)

ans = ""
for Number in 2:9
    for i in 1:10^(Int(floor(9/Number)))
        s = join(string(i * j) for j in 1:Number)
        if join(sort(collect(s))) == "123456789"
            ans = max(s, ans)
        end
    end
end

println(ans)

File: python/p_038.py
ans = ""
for Number in range(2, 9):
	for i in range(1, 10**(9 // Number)):
		s = "".join(str(i * j) for j in range(1, Number + 1))
		if "".join(sorted(s)) == "123456789":
			ans = max(s, ans)
			
print(ans)